### DISCUSSION TITLE
I am not able to use the Microsoft azure

### DISCUSSION THREAD

**Discussion by HackerAditya56** [community/community#138095] [GitHub Education]
### Hi everyone,

When I try to loging to Microsoft azure it's saying that 25-digit verification code and when I select the github option it's saying that verification of GitHub student developer pack is not completed 😔😔 please GitHub help me to find the solution

**Comment by HackerAditya56** [comment]
![Screenshot_20240908-124842_Chrome](https://github.com/user-attachments/assets/aa69b238-488f-4e7e-ac58-68beb474ea4d)

![Screenshot_20240908-124939_Chrome](https://github.com/user-attachments/assets/2b5d3625-15b2-44a4-a529-93cd41e8a7da)

**Comment by HackerAditya56** [comment]
I don't have access to my school email id 
What to do now?

**Comment by davevad93** [comment]
Hi @HackerAditya56 ,

This is a known issue that has been solved before #135739 , you have two options:

1. Contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Azure verification. 

Or

2.  Create a support ticket directly on [Azure](https://azure.microsoft.com/en-us/support/create-ticket).

By contacting support you'll be able to solve it.
